.github/workflows/test.yml

```
@@ -43,12 +50,10 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
-      - name: Install tools from asdf config
-        uses: ai/asdf-cache-action@v1
-      - name: Install dependencies
-        run: pnpm install --ignore-scripts
-      - name: Build assets
-        run: cd web && pnpm build
+      - name: Download built assets artifacts
```

```suggestion
      - name: Download web assets from test step
```

Let’s explain how made it for better readability